manifests/cli.pp

```diff
@@ -31,16 +31,6 @@
   # The jenkins cli command with required parameter(s)
   $cmd = "java -jar ${jar} -s http://localhost:${port}"
 
-  # Reload all Jenkins config from disk (only when notified)
-  exec { 'reload-jenkins':
```

Comment:
I'm curious why this has been removed, I presume it's because it is now redundant, but removing it does break API compatibility